kuma/javascript/src/active-banner.jsx

```diff
@@ -182,9 +190,48 @@ function Banner(props: BannerProps) {
 export const DEVELOPER_NEEDS_ID = 'developer_needs';
 export const SUBSCRIPTION_ID = 'subscription_banner';
 
+function SubscriptionBanner() {
+    const ga = useContext(GAProvider.context);
+    const trackAndNavigate = useTrackAndNavigate();
+
+    useEffect(() => {
+        ga('send', {
+            hitType: 'event',
+            eventCategory: CATEGORY_MONTHLY_PAYMENTS,
+            eventAction: 'CTA shown',
+            eventLabel: 'banner',
+        });
+    }, [ga]);
+
+    return (
+        <Banner
+            id={SUBSCRIPTION_ID}
+            classname="mdn-subscriptions"
+            title={gettext('Become a monthly supporter')}
+            // do not hardcode dollar amount, use CONTRIBUTION_AMOUNT_USD
+            // https://github.com/mdn/kuma/issues/6654
+            copy={gettext('Support MDN with a $5 monthly subscription')}
```

Comment:
It is kinda a bummer that we keep pushing this as now, what will happen is that translators will today get `'Support MDN with a $5 monthly subscription'` and soon it'll become `'Support MDN with a %(amount)s monthly subscription'` and that gettext will have to start over.